components/forms/Toggle/react.test.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import Toggle from './react'
+
+describe('Toggle', () => {
+  let props
+  let wrapper
+
+  beforeEach(() => {
+    wrapper = mount(<Toggle />)
+  })
+
+  describe('Should render with default properties set when nothing is passed in', () => {
+    it('Should render correctly', () => {
+      expect(wrapper).toMatchSnapshot()
+    })
+
+    it('Should render with the className of "toggle" on the initial load', () => {
+      expect(wrapper.find('.toggle').length).toBe(1)
+    })
+
+    it('Should render the toggle switch in an unchecked state', () => {
+      expect(wrapper.find('.toggle__input').props().checked).toBe(false)
+    })
+  })
+
+  describe('Should render the component when properties are passed into it and the checked state is false', () => {
+    beforeEach( () => {
+      props = {
+        onChange: jest.fn(),
+        checked: false,
+        disabled: false
+      }
+
+      wrapper = mount(<Toggle {...props} />)
+    })
+
+    it('Should render correctly', () => {
+      expect(wrapper).toMatchSnapshot()
+    })
+
+    it('Should change to a disabled state when the toggle div is clicked upon', () => {
+      wrapper.find('.toggle').simulate('click')
+      expect(props.onChange).toHaveBeenCalledTimes(1)
+      expect(wrapper.find('toggle--disabled')).toBeTruthy()
+    })
+
+    it('Should change to a disabled state when a key is pressed when inside the toggle div', () => {
+      const mockEvent = { target: { value: !props.value }}
+      wrapper.find('.toggle').prop('onKeyPress')(mockEvent)
+
+      expect(props.onChange).toHaveBeenCalledTimes(1)
+      expect(wrapper.find('toggle--disabled')).toBeTruthy()
```

Comment:
I understand what these are trying to test, but it is using the wrong class. Not sure how to check for this, but this is the style selector for showing a checked Toggle `&__input:checked + &__slider::before`